vms/platformvm/camino_helpers_test.go

```diff
@@ -48,6 +52,22 @@ var (
 	_, caminoPreFundedNodeIDs = nodeid.LoadLocalCaminoNodeKeysAndIDs(localStakingPath)
 )
 
+func defaultCaminoService(t *testing.T, camino api.Camino, utxos []api.UTXO) *CaminoService {
```

Comment:
`defaultCaminoService` is moved here from `vms/platformvm/camino_service_test.go`, because its just like other test-helper funcs.